src/reducers/entities/channels.js

```diff
@@ -102,6 +102,14 @@ function channels(state = {}, action) {
         };
     }
 
+    case SchemeTypes.RECEIVED_SCHEME_CHANNELS: {
+        const nextState = {...state};
+        for (const channel of action.data) {
+            nextState[channel.id] = channel;
+        }
+        return nextState;
+    }
```

Comment:
Since this is the same as `RECEIVED_CHANNELS` you could do this so you don't have to duplicate the code:

```javascript
        case SchemeTypes.RECEIVED_SCHEME_CHANNELS:
        case ChannelTypes.RECEIVED_CHANNELS: {
        const nextState = {...state};
        for (const channel of action.data) {
            nextState[channel.id] = channel;
        }
        return nextState;
    }
```